suites/quincy/rbd/tier-2_rbd_mirror_failback.yaml

```diff
@@ -0,0 +1,258 @@
+#===============================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_mirror_failback.yaml
+#
+# Cluster Configuration:
+#    cephci/conf/quincy/rbd/3_clusters_5_node.yaml
+#    No of Clusters : 3
+#    Each cluster configuration
+#    5-Node cluster(RHEL-8.3 and above)
+#    1 MONS, 1 MGR, 3 OSD and 1 RBD MIRROR service daemon(s)
+#     Node1 - Mon, Mgr, Installer
+#     Node2 - client
+#     Node3 - OSD
+#     Node4 - OSD
+#     Node5 - OSD, RBD Mirror
+#===============================================================================================
+tests:
+  - test:
```

Comment:
Please add this to the pipeline metadata file in scheduled pipeline for 5.3, 6.0 and 6.1 yaml files.

Reply:
@sunilangadi2 will raise a new PR for adding metadata files.